configuration/items.md

```diff
@@ -291,24 +291,18 @@ Users may add their own icons in either `png` or `svg` format in the openHAB ico
 
 The following guidelines apply to user-added icon files:
 
--   Only `png` or `svg` file formats may be used
-
--   Icon filenames may include lowercase letters, numbers and underscores (`_`)
-
--   Uppercase letters and special characters are prohibited
-
--   Hyphens (`-`) are reserved for [Dynamic Icons](#dynamic-icons) (see below)
-Example filenames:
-
--   Good: `myswitch.svg`, `power_meter.png`, `error23.svg`
-
--   Bad: `PC_Display.svg`, `power-meter.png`, `tür⇔.svg`
-
+- Only `png` or `svg` file formats may be used
+- Icon filenames may include lowercase letters, numbers and underscores (`_`)
+- Uppercase letters and special characters are prohibited
+- Hyphens (`-`) are reserved for [Dynamic Icons](#dynamic-icons) (see below)
+- Example filenames:
+  - Good: `myswitch.svg`, `power_meter.png`, `error23.svg`
+  - Bad: `PC_Display.svg`, `power-meter.png`, `tür⇔.svg`
```

Comment:
Will look better in the resulting document. I realized that after a while and will need to update the style guide